src/resources/views/manual/tutorials/qgis-getting-started.blade.php

```diff
@@ -0,0 +1,69 @@
+@extends('manual.base')
+
+@section('manual-title', 'Getting Started')
+
+@section('manual-content')
+  <div class="row">
+    <p class="lead">Introduction on Interfacing Biigle Api with QGis.</p>
+    <p>
+      QGis is an open source Geographic Information System that is used for viewing, editing and analysis of geospatial data. In order to interface Biigle with QGis, API endpoints, which respond with GeoJSON format Data, had been added. These API’s can only be accessed by Project Editor, Expert and Admins.
```

Comment:
The API endpoints can be accessed by all project members. At least this should be the case.
```suggestion
      QGis is an open source Geographic Information System that is used for viewing, editing and analysis of geospatial data. In order to interface Biigle with QGis, API endpoints, which respond with GeoJSON format Data, had been added.
```